Cargo.toml

```diff
@@ -15,4 +16,12 @@ pe = []
 vm-memory = ">=0.2.0"
 
 [dev-dependencies]
+criterion = "=0.3.0"
```

Comment:
Question, is it possible to gate this dependency on a feature? When we started using criterion in Firecracker it pretty much double the debug build time and I was wondering if we can add this dependency only when we want to run performance tests.

Reply:
It doesn't look like `cargo build` takes the `criterion` dev-dependency into account:
```
root@be984d61cd88:/linux-loader# cargo clean && cargo build  --all-features
warning: dependency (vm-memory) specified without providing a local path, Git repository, or version to use. This will be considered an error in future versions
   Compiling libc v0.2.67
   Compiling vm-memory v0.2.1
   Compiling linux-loader v0.1.0 (/linux-loader)
    Finished dev [unoptimized + debuginfo] target(s) in 3.23s
```